.editorconfig

```diff
@@ -1,7 +1,21 @@
-[*.{kt,kts}]
+root = true
 
-ktlint_disabled_rules=no-wildcard-imports,max-line-length
+[*]
+ij_continuation_indent_size = 4
 
+[*.{kt,kts}]
+ij_kotlin_code_style_defaults = KOTLIN_OFFICIAL
 # Imports must be ordered in lexicographic order without any empty lines in-between.
 # https://github.com/pinterest/ktlint/issues/1236
 ij_kotlin_imports_layout=*
+ktlint_code_style = intellij_idea
+ktlint_standard_no-wildcard-imports = disabled
+ktlint_standard_max-line-length = disabled
+
+ktlint_standard_class-naming = disabled
+ktlint_standard_comment-wrapping = disabled
+ktlint_standard_discouraged-comment-location = disabled
+ktlint_standard_function-naming = disabled
+ktlint_standard_property-naming = disabled
+ktlint_standard_trailing-comma-on-call-site = disabled
+ktlint_standard_trailing-comma-on-declaration-site = disabled
```

Comment:
These two triggered too many changes?

Reply:
I didn't want to make that change without some input. From the [Kotlin coding conventions](https://kotlinlang.org/docs/coding-conventions.html#trailing-commas):

> The Kotlin style guide encourages the use of trailing commas at the declaration site and leaves it at your discretion for the call site.

I'll have it auto-format declaration. If the same is wanted for call, I can run that too.